source/fundamentals/crud/read-operations.txt

```diff
@@ -0,0 +1,15 @@
+.. _csharp-crud-write-operations:
+
+===============
+Read Operations
+===============
+
+.. default-domain:: mongodb
+
+- :ref:`csharp-delete-guide`
```

Comment:
Issue: the TOC here is not accurate to the pages actually contained under Read Operations